### ISSUE TITLE
@-rules parsed incorrectly

### ISSUE THREAD

**Issue by mikkorantalainen** [CSSLint/csslint#211]
Example:

``` css
@viewport
{
    width: device-width;
}
```

If an @-rule is not regognized, a conforming CSS parser must ignore the whole block. If and when CSS Lint does not regognize the @viewport rule, it should say "unregognized @-rule: viewport" instead of issuing a parse error followed by a warning that "device-width" is not a valid value for width.
